packages/connection/legacy/class-jetpack-xmlrpc-server.php

```diff
@@ -100,7 +100,7 @@ public function xmlrpc_methods( $core_methods ) {
 		}
 
 		/**
-		 * Filters the XML-RPC methods available to Jetpack for unauthenticated users.
+		 * Filters the XML-RPC methods available to Jetpack for requests signed only with a blog token and without an authenticated user.
```

Comment:
Are "signed only with a blog token" and "without an authenticated user" two different things, or just one?

If it's two things, I might make that clearer by saying something like `requests signed only with a blog token and requests without an authenticated user`. If it's just one, I might make it clearer by saying something like `requests signed only with a blog token, i.e. without an authenticated user`.

Reply:
It's the same thing. I was trying to be more specific but agree it can be more confusing. I think just suppressing the last par is simpler because there is no way to sign with a blog token and to have an authenticated user. They are exclusive